examples/ts-lb-with-subnets/index.ts

```diff
@@ -10,25 +11,41 @@ const lb = new awsx.lb.ApplicationLoadBalancer("nginx-lb", {
     subnetIds: vpc.publicSubnetIds.apply((x) => x!),
 });
 
-const service = new awsx.ecs.FargateService("my-service", {
-    cluster: cluster.arn,
-    assignPublicIp: true,
-    taskDefinitionArgs: {
-        container: {
-            image: "nginx:latest",
-            name: "nginx",
-            cpu: 512,
-            memory: 128,
-            essential: true,
-            portMappings: [
-                {
-                    containerPort: 80,
-                    targetGroup: lb.defaultTargetGroup,
-                },
-            ],
+const service = new awsx.ecs.FargateService(
+    "my-service",
+    {
+        cluster: cluster.arn,
+        assignPublicIp: true,
+        taskDefinitionArgs: {
+            container: {
+                image: "nginx:latest",
+                name: "nginx",
+                cpu: 512,
+                memory: 128,
+                essential: true,
+                portMappings: [
+                    {
+                        containerPort: 80,
+                        targetGroup: lb.defaultTargetGroup,
+                    },
+                ],
+            },
         },
     },
-});
+    {
+        transformations: [
+            (args) => {
+                if (args.type === "aws:ecs/service:Service") {
```

Comment:
Ah, very interesting, so this is transforming the "invisible" resource that's not visible in the program. This is a great but not very obvious workaround.